opcua/ua/uatypes.py

```diff
@@ -341,6 +343,7 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
 
+# FIXME: This is not according to standard, check Part 3, Page 59
```

Comment:
this seems to be higher level, NodeIdType is about encoding. please remove the FIXME line